pkg/storage/bucket/swift/config.go

```diff
@@ -41,6 +44,9 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 
 // RegisterFlagsWithPrefix registers the flags for Swift storage with the provided prefix
 func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
+	f.StringVar(&cfg.ApplicationCredentialID, prefix+"swift.application-credential-id", "", "OpenStack Swift application credential id")
```

Comment:
In addition to adding the configuration here, the values need to be passed to the Thanos configuration object created [here](https://github.com/grafana/mimir/blob/f52911d917c8c52e0da6a59348a64dd7f7622072/pkg/storage/bucket/swift/bucket_client.go).

Reply:
Ok I've added those.